frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -0,0 +1,56 @@
+# Contributing Data
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
```

Comment:
_We do not allow submission_ -> _We do not accept datasets_

Please be consistent about _dataset_ vs _data set_.